tdex/docker-compose.yml

```diff
@@ -6,31 +6,52 @@ services:
       APP_HOST: tdex_caddy_1
       APP_PORT: 8080
 
+  oceand:
+    image: ghcr.io/vulpemventures/oceand:v0.1.17@sha256:9a7e1b8df4d7907ad66e3475c5a5f357d73c49df96214c00eba121f2aea9a50d
+    tty: true
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/ocean-data:/home/ocean/.oceand
+    environment:
+      OCEAN_LOG_LEVEL: 5
+      OCEAN_DB_TYPE: "badger"
+      OCEAN_NO_TLS: "true"
+      OCEAN_NO_PROFILER: "true"
+      OCEAN_UTXO_EXPIRY_DURATION_IN_SECONDS: 240
+
   tdexd:
-    image: ghcr.io/tdex-network/tdexd:v0.9.1@sha256:5078897b9a505fc3dfac5b3d51537150941c075f5279ae81de1c6b1c21112544
+    image: ghcr.io/tdex-network/tdexd:v1.0.0@sha256:6e6733015dceeab41db4780b5e99313d5bbbc5dc490800ee3b5a326a8b65b92d
     tty: true
     restart: on-failure
+    depends_on:
+      - oceand
     stop_grace_period: 1m
     volumes:
       - ${APP_DATA_DIR}/tdex-data:/home/tdex/.tdex-daemon
+    ports:
+     - ${APP_TDEX_PORT}:${APP_TDEX_PORT}
```

Comment:
we don't need to map ports here, as Docker's internal DNS resolution will allow the containers in this compose file to connect to each other. Unless you are trying to expose this port on the host so that it is reachable externally?

Reply:
Yeah the idea is to make this service accessible from the outside world.

Reply:
ah okay, so in that case we need to change `${APP_TDEX_PORT}` to something other than `9090` because this port has been taken by Urbit Bitcoin Connector, meaning if a user is running both TDEX and Urbit Bitcoin Connector there will be a port clash on the host and one of the apps will break:
https://github.com/getumbrel/umbrel-apps/blob/5ec41a7ae69eb202cf43533d8749a105a4444477/urbit-bitcoin-connector/umbrel-app.yml#L29

Reply:
ah okay, so in that case we need to change `${APP_TDEX_PORT}` to something other than `9090` because this port has been taken by Urbit Bitcoin Connector, meaning if a user is running both TDEX and Urbit Bitcoin Connector there will be a port clash on the host and one of the apps will break:
https://github.com/getumbrel/umbrel-apps/blob/5ec41a7ae69eb202cf43533d8749a105a4444477/urbit-bitcoin-connector/umbrel-app.yml#L29

Reply:
it shouldn't be a problem at all, do you suggest any port number?

Reply:
what about `9000`?

Reply:
Hi @altafan , port 9000 is currently occupied by portainer. To avoid conflicts, please consider using 9002, which is the closest available option. Before selecting a port, it's a good practice to search within this repository to ensure it doesn't overlap with existing port assignments.